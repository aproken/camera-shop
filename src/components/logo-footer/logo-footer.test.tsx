import { render, RenderResult } from '@testing-library/react';
import LogoFooter from './logo-footer';
import { BrowserRouter as Router } from 'react-router-dom';


describe('Компонент LogoFooter', () => {
  it('Тестирование компонента Footer', () => {
    const { container }: RenderResult = render(
      <Router>
        <LogoFooter />
      </Router>
    );
    expect(container).toMatchSnapshot();
  });
});
