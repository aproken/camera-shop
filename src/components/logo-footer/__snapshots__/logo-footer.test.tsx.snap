// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: LogoFooter Тестирование компонента Footer 1`] = `
<div>
  <a
    aria-label="Переход на главную"
    class="footer__logo"
    href="/"
  >
    <svg
      aria-hidden="true"
      height="36"
      width="100"
    >
      <use
        xlink:href="#icon-logo-mono"
      />
    </svg>
  </a>
</div>
`;

exports[`Компонент LogoFooter Тестирование компонента Footer 1`] = `
<div>
  <a
    aria-label="Переход на главную"
    class="footer__logo"
    href="/"
  >
    <svg
      aria-hidden="true"
      height="36"
      width="100"
    >
      <use
        xlink:href="#icon-logo-mono"
      />
    </svg>
  </a>
</div>
`;
